backend/datasource/manage.go

```diff
@@ -18,9 +18,6 @@ type ManageOpts struct {
 
 	// TracingOpts contains settings for tracing setup.
 	TracingOpts tracing.Opts
-
-	// Stateless admission handler
-	AdmissionHandler backend.AdmissionHandler
 }
```

Comment:
This is unused now so deleting it. This forced plugins to define the AdmissionHandler outside of the `Datasource` struct since it's not available at the moment of using these Opts when using auto-management.